wal: add page (#3381)
tldr-pages/tldr: pages/common/pywal.md

> # pywal

Reviewer: Perhaps we could rename this page to `wal` please? tldr pages should have a title of the command name they are about. We should rename the file too.

Author: Can do! I just wasn't sure which name to use, 'pywal' or, 'wal'.

Reviewer: Yeah, we use the name of the command, because through the tldr CLI you're most likely to do `tldr {{command_name}}`.

> > A tool to create colorschemes based on dominant colors of a wallpaper.

Reviewer: Perhaps this will read better?
```suggestion
> A tool to create color schemes based on the dominant colors of a wallpaper.
```

> > Read more: https://github.com/dylanaraps/pywal.

Reviewer: Perhaps this will read better?
```suggestion
> More information: <https://github.com/dylanaraps/pywal>.
```

> - Preview colorscheme:

Reviewer: Perhaps this will read better?
```suggestion
- Preview color scheme:
```

`wal --preview image.png`

> - Create colorscheme:

Reviewer: Perhaps this will read better?
```suggestion
- Create color scheme:
```

> `wal -i image.png`

Reviewer: Is there a long form of the `-i` option here? In tldr pages we prefer to use the long form option where possible to aid clarity. The same goes for the later examples too.

Author: There is no long form of the `-i` option, along with all the other examples. `--preview` is the only long form option in that example.

> - Create light colorscheme:

Reviewer: Perhaps this will read better?
```suggestion
- Create a light color scheme:
```

`wal -il image.png`

> - Skip setting wallpaper:

Reviewer: Perhaps this will read better?
```suggestion
- Skip setting the desktop wallpaper:
```

`wal -in image.png`

> - Skip setting terminal colors:

Reviewer: Perhaps this will read better?
```suggestion
- Skip setting the terminal colors:
```

`wal -is image.png`

> - Restore colorscheme and wallpaper:

Reviewer: Perhaps this will read better?
```suggestion
- Restore the original  color scheme and wallpaper:
```

Author: By "restore color scheme", it reapplies the color scheme that `wal` generated previously. E.g. in my [bspwmrc](https://github.com/kaewhyes/dotfiles/blob/master/bspwm/bspwmrc#L9), I have `wal -R`, which re-applies the color scheme, setting my polybar colors, border colors, etc.

Author: Would
```suggestion
- Restore colorscheme and wallpaper:
+ Restore generated color scheme and wallpaper:
```
sound better?

Author: Would "Restore the generated color scheme and wallpaper" sound better?

Reviewer: Ah, I see. Yeah, something like that. What about `Restore the previously generated color scheme and wallpaper`?

Author: Instead of `Restore the original  color scheme and wallpaper:`, writing, `Restore the previously generated color scheme and wallpaper`.

Reviewer: Yeah, something like that @kaewhyes
